src/components/form-elements/_form-elements.scss

```diff
@@ -95,30 +93,27 @@ $includeHtml: false !default;
       }
     }
 
-    &:before {
-      $iconSize: $selectFontSize + $iconBoostValue;
-      font-size: $iconSize;
-      font-weight: bold;
+    &__icon {
       position: absolute;
-      right: $selectHeight / 2 - $iconSize / 2;
-      top: 50%;
+      right: ($selectHeight / 4 + rhythm(0.334)) - $selectIconSize / 2;
```

Comment:
Its an overkill to have variables for numbers in sass :)

Just change it to `0.333` as `round(0.(3)) ~= 0.333`